Things.md

```diff
@@ -114,4 +116,31 @@ FSM.setOpacity(thing, 1);
 
 // Sets a Thing's opacity to 0 (transparent). 
 FSM.setOpacity(thing, 0);
-```
\ No newline at end of file
+```
+
+# Adding Things  
+You're able to add Things to the game via `addThing`, specifying the `x` and `y` positions (again this is relative to the top left corner of the screen). The `x` is where the Thing's `left` bound will be and the `y` is where the Thing's `top` bound will be. You can add it using the class title for the Thing:
+```javascript
+// Adds a Thing to the game at position (1,2) using the class name.
+FSM.addThing("CastleFireball", 1, 2);
+
+// Adds a Thing next to (to the right of) another Thing.
+FSM.addThing("CastleFireball", otherThing.right, otherThing.top);
+
+// Adds a Thing below another Thing.
+FSM.addThing("CastleFireball", otherThing.left, otherThing.bottom);
+```
+
+# TimeHandlr  
+TimeHandlr is a module used for setting timed events. It's a more flexible alternative to javascript's `setTimeout` and `setInterval`. It also handles the progression of in-game time using `handleEvents`, which steps time forward one step and calls all events that are supposed to occur at this time. You can also use it to add your own events:
```

Comment:
This is a pretty mediocre explanation. Please redo it.

* JavaScript, not javascript.
* No consumer of a GameStartr game will ever interact with `handleEvents`.
* "steps time forward one step" is not explained.